CONTRIBUTING.md

```diff
@@ -60,25 +62,34 @@ This section describes how to build and run the Botkube from the source code.
      docker tag ghcr.io/kubeshop/botkube:v9.99.9-dev-amd64 <your_account>/botkube:v9.99.9-dev
      docker push <your_account>/botkube:v9.99.9-dev
      ```
-     Where `<your_account>` is Docker hub account to which you can push the image.
+     Where `<your_account>` is Docker hub or any other registry provider account to which you can push the image.
+
+2. Ensure you have a Slack installation already setup:
 
-2. Deploy the newly created image in your cluster:
+   Please refer to the [Slack installation instructions](https://botkube.io/docs/installation/socketslack/) to ensure you have a valid integration.
 
+3. Deploy the newly created image in your cluster:
+   
    ```sh
    helm install botkube --namespace botkube --create-namespace \
-   --set communications.slack.enabled=true \
-   --set communications.slack.channel=<SLACK_CHANNEL_NAME> \
-   --set communications.slack.token=<SLACK_API_TOKEN_FOR_THE_BOT> \
-   --set settings.clusterName=<CLUSTER_NAME> \
-   --set settings.kubectl.enabled=<ALLOW_KUBECTL> \
-   --set image.registry=<image_registry e.g. docker.io> \
+   --set communications.default-group.socketSlack.enabled=true \
+   --set communications.default-group.socketSlack.channels.default.name=<SLACK_CHANNEL_NAME> \
+   --set communications.default-group.socketSlack.appToken=<SLACK_API_APP_TOKEN> \
+   --set communications.default-group.socketSlack.botToken=<SLACK_API_BOT_TOKEN> \
+   --set settings.clusterName=${CLUSTER_NAME} \
```

Comment:
To be consistent with the rest of the flags:
```suggestion
   --set settings.clusterName=<CLUSTER_NAME> \
```

But please read my different comment as probably we could get rid of this line at all 👍